README.md

```diff
@@ -78,4 +77,29 @@ $ docker compose exec counterparty.local envoy createuser -e [email] -r admin
 
 You can access the counterparty at [http://localhost:9000](http://localhost:9000) or at [http://counterparty.local:9000](http://counterparty.local:9000) if you have edited your hosts file.
 
-> **NOTE**: Due to the way cookie domains work with the credentials, you can only be logged into either the envoy development node or the counterparty development node at the same time. It's annoying, but you'll have to login again when switching between nodes unfortunately.
\ No newline at end of file
+> **NOTE**: Due to the way cookie domains work with the credentials, you can only be logged into either the envoy development node or the counterparty development node at the same time. It's annoying, but you'll have to login again when switching between nodes unfortunately.
+
+
+## Envoy Implementation Options
+
+| Open Source                                                                                                         | One-Time Setup                                                                                                                                    | Managed Service                                                                                                                                                 |
```

Comment:
codebeautify.org/markdown-table-generator was used to generate the markdown tables.

Reply:
I use that one too :-)